frontend/src/ui-components/AcmCharts/AcmDonutChart/AcmDonutChart.tsx

```diff
@@ -137,3 +137,23 @@ export function AcmDonutChart(props: {
         </Card>
     )
 }
+
+export const colorThemes = {
+    failureWarningSuccess: [
+        'var(--pf-chart-color-red-100)',
+        'var(--pf-global--warning-color--100)',
+        'var(--pf-chart-color-black-100)',
+    ],
+    failureSuccess: ['var(--pf-chart-color-red-100)', 'var(--pf-chart-color-black-100)'],
+    failurePendingSuccess: [
+        'var(--pf-chart-color-red-100)',
+        'var(--pf-chart-color-gold-300)',
+        'var(--pf-chart-color-black-100)',
+    ],
+    dangerImportantModerateInfo: [
+        'var(--pf-chart-color-red-100)',
+        'var(--pf-global--warning-color--100)',
+        'var(--pf-chart-color-gold-300)',
+        'var(--pf-global--info-color--100)',
+    ],
+}
```

Comment:
Much better! There is still repetition in here I'd like to see factored out. ie: introduce a constant for `failure`, `success`, and so on.

Also, please double-check the colours against the latest from @heyethankim 
- should warning be --pf-chart-color-gold-300 for "Pending" or any intermediate status?
- should the insights colors be called Critical, Important, Moderate, Low? The prescribed colors should all be from pf-chart I think?

Reply:
> * should warning be --pf-chart-color-gold-300 for "Pending" or any intermediate status? **YES**

Reply:
> * should the insights colors be called Critical, Important, Moderate, Low? The prescribed colors should all be from pf-chart I think?

This is what I shared. If you could swap the "global colors" with the "chart colors," that will be great.  
![image](https://user-images.githubusercontent.com/84546841/206221781-9dcf32b7-e76e-432d-ada9-7b6a3b6cd873.png)